source/connect/tls.txt

```diff
@@ -0,0 +1,241 @@
+  option to ``true``
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/connect/tls-connection-str.c
+         :language: c
+         :start-after: start-connect-to-atlas
+         :end-before: end-connect-to-atlas
+         :dedent:
+
+   .. tab:: MongoC URI Options
+      :tabid: mongocurioptions
+
+      .. literalinclude:: /includes/connect/tls-uri-options.c
+         :language: c
+         :start-after: start-connect
+         :end-before: end-connect
+         :dedent:
+
+.. tip::
+   
+   If your connection string includes the ``+srv`` modification, which specifies the
+   SRV connection format, TLS is enabled on your connection by default.
+
+   To learn more about the SRV connection format, see
+   :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
+   in the {+mdb-server+} documentation.
+
+.. _c-specify-ca-file:
+
+Specify a CA File
+------------------
+
+When MongoDB is started with TLS enabled, it will by default require the client to provide
+a client certificate issued by a certificate authority specified by `--tlsCAFile`, or an authority
+trusted by the native certificate store in use on the server.
+
+To provide the client certificate, set the `tlsCertificateKeyFile` in the URI to a PEM armored
+certificate file:
+
+.. literalinclude:: /includes/connect/tls-ca-file.c
+         :language: c
+         :start-after: start-connect
+         :end-before: end-connect
+         :dedent:
+
+.. _c-certificate-revocation:
+
+Server Certificate Verification
+-------------------------------
+
+The MongoDB C Driver will automatically verify the validity of the server certificate, such as issued
+by configured Certificate Authority, hostname validation, and expiration.
+
+To overwrite this behavior, it is possible to disable hostname validation, OCSP endpoint revocation checking,
+revocation checking entirely, and allow invalid certificates.
+
+This behavior is controlled using the ``tlsAllowInvalidHostnames``, ``tlsDisableOCSPEndpointCheck``,
+``tlsDisableCertificateRevocationCheck``, and ``tlsAllowInvalidCertificates`` options. By default,
+all are set to ``false``.
+
+It is not recommended to change these defaults as it exposes the client to *Man In The Middle* attacks (when
+``tlsAllowInvalidHostnames`` is set), invalid certificates (when ``tlsAllowInvalidCertificates`` is set), or
+potentially revoked certificates (when ``tlsDisableOCSPEndpointCheck`` or ``tlsDisableCertificateRevocationCheck``
+are set).
+
+Supported Libraries
+-------------------
+
+By default, libmongoc will attempt to find a supported TLS library and enable TLS support. This is controlled
+by the cmake flag ``ENABLE_SSL``, which is set to ``AUTO`` by default. Valid values are:
+
+- ``AUTO`` the default behavior. Link to the system's native TLS library, or attempt to find OpenSSL.
```

Comment:
S: Usually we use colons as [separators in lists](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items)

```suggestion
- ``AUTO``: Links to the system's native TLS library, or attempts to find OpenSSL. This is the default value.
```